components/tools/OmeroWeb/omeroweb/webclient/webclient_gateway.py

```diff
@@ -1382,8 +1383,7 @@ def updateGroup(self, group, name, permissions, owners=list(),
         # Should we update updateGroup so this would be atomic?
         admin_serv.updateGroup(up_gr)
         if permissions is not None:
-            logger.warning("WARNING: changePermissions was called!!!")
```

Comment:
I'd guess that it's *not* happening, but the warning was essentially documentation about what can go wrong. Not sure if we should just remove this branch entirely. @aleksandra-tarkowska may know more.